app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -462,6 +464,17 @@ def test_serializer_outputs_expected_schema_for_addon_experiment(self):
 
         self.assertEqual(serializer.data["experimenter_slug"], experiment.slug)
 
+    def test_serializer_outputs_expect_schema_for_branched_addon(self):
+        experiment = ExperimentFactory.create(
+            firefox_min_version="70.0", type=Experiment.TYPE_ADDON
+        )
+        serializer = ExperimentRecipeSerializer(experiment)
+        self.assertEqual(serializer.data["action_name"], "opt-out-study")
+        self.assertEqual(
+            serializer.data["arguments"],
+            ExperimentRecipeBranchedArgumentsSerializer(experiment).data,
```

Comment:
If we don't test the actual individual fields anywhere we could just remove all the fields from the serializer and all the tests would still pass but it would break in prod?

Reply:
I'd still like to see the fields for this serializer unpacked in this test.